cogs/tags.py

```diff
@@ -107,115 +170,338 @@ async def all(self, ctx: commands.Context):
+                text=f"Deleted By: {ctx.author}", icon_url=ctx.author.avatar_url
+            )
+        )
 
     @tag.command()
     @commands.cooldown(1, 1, commands.BucketType.user)
     async def search(self, ctx, *, term: str):
         """Search for a tag given a search term. PostgreSQL syntax must be used for the search."""
-        query = """SELECT name FROM tags WHERE guild_id = $1 AND name LIKE $2 LIMIT 10"""
+        query = (
+            """SELECT name FROM tags WHERE guild_id = $1 AND name LIKE $2 LIMIT 10"""
+        )
         records = await self.bot.db.fetch(query, ctx.guild.id, term)
 
         if not records:
-            return await ctx.send("No tags found that has the term in it's name", delete_after=10)
+            return await ctx.send(
+                "No tags found that has the term in it's name", delete_after=10
+            )
         count = "Maximum of 10" if len(records) == 10 else len(records)
         records = "\n".join([record["name"] for record in records])
 
-        await ctx.send(f"**{count} tags found with search term on this server.**```\n{records}\n```")
+        await ctx.send(
+            f"**{count} tags found with search term on this server.**```\n{records}\n```"
+        )
 
     @tag.command()
     @is_engineer_check()
-    async def rename(self, ctx, name: lambda inp: inp.lower(), *, new_name: lambda inp: inp.lower()):
+    async def rename(
+        self, ctx, name: lambda inp: inp.lower(), *, new_name: lambda inp: inp.lower()
+    ):
         """Rename a tag."""
-
+        name = await commands.clean_content().convert(ctx=ctx, argument=name)
         new_name = await commands.clean_content().convert(ctx=ctx, argument=new_name)
 
         tag = await self.bot.db.get_tag(guild_id=ctx.guild.id, name=name)
 
         if tag is None:
             await ctx.message.delete(delay=10.0)
-            message = await ctx.send('Could not find a tag with that name.')
+            message = await ctx.send("Could not find a tag with that name.")
             return await message.delete(delay=10.0)
 
         if tag.creator_id != ctx.author.id:
             if not is_admin(ctx.author):
-                return await ctx.send('You don\'t have permission to do that.')
-        
+                return await ctx.send("You don't have permission to do that.")
+
         try:
             await tag.rename(new_name=new_name)
         except asyncpg.UniqueViolationError:
             return await ctx.send("A tag with that name already exists.")
 
-        await ctx.send('You have successfully renamed your tag.')
+        await ctx.send("You have successfully renamed your tag.")
 
     @tag.command()
     @is_engineer_check()
     async def append(self, ctx, name: lambda inp: inp.lower(), *, text: str):
         """Append some content to the end of a tag"""
+        name = await commands.clean_content().convert(ctx=ctx, argument=name)
         text = await commands.clean_content().convert(ctx=ctx, argument=text)
```

Comment:
I think `name: commands.clean_content, *, text: commands.clean_content` then `name = name.lower()` would be clearer than invoking the converter for clean content.